app/Services/Entity/MultiEditingService.php

```diff
@@ -86,9 +101,21 @@ public function edit(): self
      */
     public function finish(): self
     {
+        if ($this->model->getTable() == 'entity_notes') {
```

Comment:
Same here, use `instanceof` instead of testing the table name which can be changed